cloudbrain/rt_server/rt_server.py

```diff
@@ -118,6 +129,10 @@ def on_close(self):
     def send_heartbeat(self):
         self.broadcast(self.clients, 'message')
 
+    def metric_exists(self, device_id, device_name, metric):
+        return (self.subscribers.has_key(device_id)
+                and self.subscribers[device_id].has_key(device_name)
+                and self.subscribers[device_id][device_name].has_key(metric))
 
 class MockHandler(RequestHandler):
     """Just a mock page to test it out..."""
```

Comment:
A couple of things about the rest of this file:

* Move the `See: http:// ...` to a doc sting and explain some more what this class does.
```
 class TornadoSubscriber(object):
     """ See: https://pika.readthedocs.org/en/0.9.14/examples/tornado_consumer.html """ 
```

There is a problem in how the params are initialized here. The rabbitMQ address is harcorded: `self.host = RABBITMQ_ADDRESS` Should be: 
```
     def __init__(self, callback, device_name, device_id, rabbitmq_address, metric_name)
         self.callback = callback
         self.device_name = device_name
         self.device_id = device_id
         self.host = rabbitmq_address
         self.metric_name = metric_name
 
         self.connection = None
         self.channel = None
         self.queue_name = None
```

Re-indent correct:
```
     def connect(self):
         credentials = pika.PlainCredentials('cloudbrain', 'cloudbrain')
         self.connection = pika.adapters.tornado_connection.TornadoConnection(
             pika.ConnectionParameters(
                 host=self.host, credentials=credentials),
             self.on_connected,
             stop_ioloop_on_close=False,
             custom_ioloop=IOLoop.instance())
 
```

Replace print statements by: 
```
_LOGGER.info( "Real-time data server running at http://localhost: {}".format( SERVER_PORT))
```